wcivf/apps/elections/templates/elections/includes/_previous_party_affiliations.html

```diff
@@ -0,0 +1,11 @@
+{% load static %}
+{% load i18n %}
+
+<i>
+    <p>Previous Party Affiliations</p>
+    {% for previous_party in person_post.previous_party_affiliations.all %}
+        <ul>
```

Comment:
Untesed, does it work to show this inline?

`Previous Party Affiliations: {% for previous_party in person_post.previous_party_affiliations.all %}{{ previous_party.party_name }}{% if not forloop.last %}, {% endif %}{% endfor %}`